packages/ruleset-migrator/src/validation/schema.ts

```diff
@@ -4,6 +4,41 @@ const schema = {
   $schema: 'http://json-schema.org/draft-07/schema#',
   type: 'object',
   properties: {
+    aliases: {
```

Comment:
A less strict variant of https://github.com/stoplightio/spectral/blob/develop/packages/core/src/meta/ruleset.schema.json#L164#L178
with added support for multiple targets